course.py

```diff
@@ -144,7 +144,11 @@ def course_info(course_id, year):
     all_years = db.session.query(Course).filter_by(id=course.id).distinct(Course.year).order_by(
         Course.year.desc()).all()
 
-    return render_template('course_info.html', course=course, all_years=all_years, current_year=year)
+    evaluations = db.session.query(Evaluation).filter(Evaluation.course_id == course_id).order_by(
```

Comment:
You should be able to specify the `order_by` directive directly in the relationship : https://docs.sqlalchemy.org/en/20/orm/relationship_api.html#sqlalchemy.orm.relationship.params.order_by